### DISCUSSION TITLE
How to ignore tabs

### DISCUSSION THREAD

**Discussion by leolux** [WinMerge/winmerge#655] [Q&A]
Currently it is possible to ignore spaces but it does not ignore tabs. How to ignore all whitespaces?

**Comment by sdottaka** [comment]
If 'Ignore change' and 'Ignore all' in the Whitespaces group of the Compare/General category in the Options dialog do not ignore the difference in the number of TAB characters, that's a bug.

If that doesn't work, could you please show us the files you tried to compare?
